pkg/app/web/src/components/application-count/index.tsx

```diff
@@ -32,12 +31,19 @@ const useStyles = makeStyles((theme) => ({
 export interface ApplicationCountProps {
   totalCount: number;
   disabledCount: number;
-  kind: ApplicationKind;
+  kind: string;
```

Comment:
nits, not sure the point of changing the type of `kind` from enum `ApplicationKind` to `string` 🤔

Reply:
https://github.com/pipe-cd/pipe/pull/1730/files/329239fc425c7614d22031d330e0ea7ffba04026#diff-0efde055bddb2f8b7015e7b2cdbbf5f83ad0cb0a9ade472851219cd14bea75e3R55-R58
```ts
            <ApplicationCount
              key={kindName}
              kind={kindName}
              totalCount={
```

This is because `string` is easier to pass when actually using this component.
It can also reduce dependency.

Reply:
Thank you for clarifying it 🙏 in that case, my suggestion is to rename this `ApplicationCount.kind` as `ApplicationCount.kindName` to avoid confusing 😅 this `kind` here makes it looks like it should be of type `ApplicationKind` as previously. How do you think about it? 😄

Reply:
@khanhtc1202 You're right! It seems better to change this.
Thank you! 👍